cinema/models.py

```diff
@@ -64,8 +78,8 @@ def __str__(self):
 class Order(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
-    )
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE)
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue